arch/Dockerfile

```diff
@@ -1,22 +1,22 @@
 # build filter using rust toolchain
-FROM rust:1.80.0 as builder
-RUN rustup -v target add wasm32-wasi
+FROM rust:1.82.0 as builder
+RUN rustup -v target add wasm32-wasip1
 WORKDIR /arch
 COPY crates .
 
-RUN cd prompt_gateway && cargo build --release --target wasm32-wasi
-RUN cd llm_gateway && cargo build --release --target wasm32-wasi
+RUN cd prompt_gateway && cargo build --release --target wasm32-wasip1
+RUN cd llm_gateway && cargo build --release --target wasm32-wasip1
 
 # copy built filter into envoy image
-FROM envoyproxy/envoy:v1.31-latest as envoy
+FROM envoyproxy/envoy:v1.32-latest as envoy
 
 #Build config generator, so that we have a single build image for both Rust and Python
-FROM python:3-slim as arch
+FROM python:3.13-slim as arch
```

Comment:
Do we want to go to python 3.13. Isn't 3.12 the more stable version?

Reply:
using python:3.12 all across now